src/screens/Movie/Movie.tsx

```diff
@@ -157,7 +163,7 @@ const Movie = ({ match, location }: RouteComponentProps<MovieRouteParams>): JSX.
         {playlist ? (
           <>
             <div className={styles.related}>
-              <h3>{playlist.title}</h3>
+              <h3>{isPlaylistPlaceholderData || playlistLoading ? 'Loading' : playlist.title}</h3>
```

Comment:
This change prevents a layout shift while loading the playlist data.  `Loading` should be translated